testomania/src/main/java/com/earth/testomania/quiz_categories/usecase/GetQuizUseCase.kt

```diff
@@ -60,9 +59,8 @@ class GetQuizListUseCase @Inject constructor(
     )
 
     private fun logProblematicQuiz(quiz: Quiz) = log(
-        "Problematic Quiz (Without correct answer) from: $QUIZ_API_BASE_URL$QUIZ_API_PATH" +
+        "Problematic Quiz (Without correct answer) from:" + //TODO //$QUIZ_API_BASE_URL$QUIZ_API_PATH" +
```

Comment:
what u want to do here?

Reply:
Aaa yes, we need also to generalize this. I want to log an event in our analytics if problematic Quiz appears to also forward info to according API to also improve their products, we can be partners or something like that

Reply:
Fixed it